### PR TITLE
add option for an already open calendar

### PR DESCRIPTION
**Pull request by ElSiham**

### CODE REVIEW

File: src/components/DatePicker/index.vue [hunk 1]
```diff
@@ -95,7 +98,7 @@
           />
           <button
             type="button"
-            class="datepicker__month-button datepicker__month-button--next -hide-up-to-tablet"
+            class="[ datepicker__month-button datepicker__month-button--next -hide-up-to-tablet"
```
Comment (joffreyBerrier) [inline src/components/DatePicker/index.vue:101]: delete the `[`

File: src/components/Day.vue [hunk 2]
```diff
@@ -5,6 +5,7 @@
       v-html="tooltipMessageDisplay"
       v-if="showTooltip && options.hoveringTooltip"
     />
+
```
Comment (joffreyBerrier) [inline src/components/Day.vue:8]: Remove this empty line

**Comment by sbourdon13** [comment]
is it normal to have the calendar icon on the top left ? 
image

File: src/App.vue [hunk 3]
```diff
@@ -2,6 +2,44 @@
...
+          :showYear="true"
+          :firstDayOfWeek="1"
+          :disableCheckoutOnCheckin="true"
+          :halfDay="true"
+          @handleCheckIncheckOutHalfDay="handleCheckIncheckOutHalfDay"
+        />
+      </div>
       <div class="box">
         <h3>
           Show calendar on the right
```
Comment (sbourdon13) [inline src/App.vue:45]: hmm this h3 disappeared with the always visible calendar

File: src/components/DatePicker/index.scss [hunk 4]
```diff
@@ -146,6 +146,36 @@ $extra-small-screen: '(max-width: 23em)';
     }
   }
 
+  &__fullview {
+    background: none;
+    height: 0;
```
Comment (sbourdon13) [inline src/components/DatePicker/index.scss:151]: why height: 0 and not height: auto ?

**Pull request merged**